CMakeTests/FindALSA.cmake

```diff
@@ -47,11 +47,13 @@ FIND_PACKAGE_HANDLE_STANDARD_ARGS(ALSA
 if(ALSA_FOUND)
   set( ALSA_LIBRARIES ${ALSA_LIBRARY} )
   set( ALSA_INCLUDE_DIRS ${ALSA_INCLUDE_DIR} )
-  add_library(ALSA::ALSA UNKNOWN IMPORTED)
-  set_target_properties(ALSA::ALSA PROPERTIES
-    IMPORTED_LOCATION ${ALSA_LIBRARY}
-    INTERFACE_INCLUDE_DIRECTORIES ${ALSA_INCLUDE_DIR}
-  )
+  if(NOT TARGET ALSA::ALSA)
```

Comment:
My patch to it was adding the imported target, and I forgot to prevent redefining it, I'm just fixing that.
And I haven't forgotten about sending a patch upstream!